src/components/docs-page/index.tsx

```diff
@@ -0,0 +1,28 @@
+import { ReactElement } from 'react'
+import { MDXRemote } from 'next-mdx-remote'
+import defaultMdxComponents from '@hashicorp/platform-docs-mdx'
+import Placement from 'components/author-primitives/shared/placement-table'
+import NestedNode from 'components/author-primitives/waypoint/nested-node'
+import makeDocsAnchor from 'components/docs-anchor'
+
+/**
+ * TODO: we may find that we want to allow additional components to be passed in
+ * as a prop in the future
+ */
+const DocsPage = (props): ReactElement => {
+  /**
+   * TODO: this will eventually be configurable by all products, not just waypoint
+   */
+  const additionalComponents = defaultMdxComponents({
+    product: 'waypoint',
+    additionalComponents: {
+      Placement,
+      NestedNode,
+      a: makeDocsAnchor('waypoint', ['docs', 'plugins', 'commands']),
+    },
+  })
```

Comment:
Can we move this out of the component function body for now? Just to ensure we aren't re-creating components during render, which might cause React to re-mount instead of re-render these components. Alternatively we could wrap it in a `useMemo` call!

Reply:
Moving it out seems easiest for the time being since we aren't currently passing any more components through the props. I'll do that now!

Reply:
Done! ✅ 
https://github.com/hashicorp/dev-portal/pull/48/commits/d3b71cb88e6f348a9e6a039f5c35f2dfdb4c104b